lib/puppet/indirector/ssl_file.rb

```diff
@@ -112,6 +112,7 @@ def search(request)
 
   def create_model(name, path)
     result = model.new(name)
+    # calls Puppet::SSL::Base#read in most circumstances
```

Comment:
In most circumstances? This comment is vaguely alarming. Why would it not be called? In case a subclass overrides it?